rush.json

```diff
@@ -314,7 +314,7 @@
     /**
      * The list of shell commands to run after the Rush installation finishes
      */
-    "postRushInstall": [],
+    "postRushInstall": ["npm i -g --save-dev prettier || exit $?"],
```

Comment:
This seems like a slightly odd combination of flags.

Reply:
would you think 'npm install -g prettier' is better?

Reply:
The `--save-dev` and `--global` shouldn't be combined, so yes, that'd be better.

Reply:
A repo shouldn't be installing something globally. We also need to ensure that a consistent version is installed.